h/services/user_signup.py

```diff
@@ -2,11 +2,16 @@
 
 from __future__ import unicode_literals
 
+import logging
 from functools import partial
+from sqlalchemy.exc import IntegrityError
```

Comment:
Minor nit: There should be a new line between the standard lib imports and third party lib imports.